egs2/an4/asr1/local/data.sh

```diff
@@ -29,6 +29,8 @@ fi
 
 . ./path.sh
 . ./cmd.sh
+a=${train_cmd}
```

Comment:
Is it for debugging or you intend to do it?
It's better to avoid `a` as a variable name.

Reply:
Oh, sorry, this file was mistakenly uploaded.